test/puppetlabs/puppetdb/http/server_test.clj

```diff
@@ -17,7 +17,7 @@
   [version versions]
 
   (testing "provides a useful error message"
-    (let [endpoint (str "/" (name version) "/nodes")
```

Comment:
This test was checking error messages, relying on nodes only accepting GET. I flipped it to reports so that the test still works.